keyboards/idank/sweep/info.json

```diff
@@ -0,0 +1,80 @@
+{
+    "url": "https://github.com/idank/keyboards/tree/main/sweep",
+    "maintainer": "idank",
+    "keyboard_name": "Ferris sweep",
+    "manufacturer": "idank",
+    "development_board": "promicro",
+    "usb": {
+        "vid": "0xC2AB",
+        "pid": "0x3939",
```

Comment:
Please do not duplicate the device IDs of `ferris/sweep` for a different PCB. Use a unique VID and PID and pick a different unique name instead of "Ferris Sweep" to avoid confusion with the existing one.

Reply:
done.